TFT/src/User/API/Printing.c

```diff
@@ -405,7 +406,6 @@ void printStart(FIL * file, uint32_t size)
   // always clean infoPrinting first and then set the needed attributes
   memset(&infoPrinting, 0, sizeof(PRINTING));
 
-  // we assume infoPrinting is clean, so we need to set only the needed attributes
```

Comment:
please, do not remove comments like this. It is simply a remind that infoPrinting was reset previously and that the other fields are so already set to 0

Reply:
You must be joking. Check the line above the removed comment. Also check the comment above the mentioned line.